pom.xml

```diff
@@ -551,6 +551,30 @@
                     <autoReleaseAfterClose>false</autoReleaseAfterClose>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>io.github.git-commit-id</groupId>
+                <artifactId>git-commit-id-maven-plugin</artifactId>
+                <version>4.9.9</version>
+                <executions>
+                    <execution>
+                        <id>get-the-git-infos</id>
```

Comment:
I would like to ask if this execution still works when being used as a maven dependency?

Reply:
Yes, the git commit hash is get in compile phase.